### PR TITLE
Milestone 2 - Redux Setup

### PR DESCRIPTION
**Pull request by omar-labana**
Highlights on this pull request:
- Prepare the directory structure
- Implement the App, Book, BookList, and  BookForm components
- Define basic Redux actions and reducers
- Initialize the Redux store

### CODE REVIEW

File: src/components/App.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import BooksForm from '../containers/BooksForm';
+import BooksList from '../containers/BooksList';
+
+const App = () => (
+  <main className="">
+    <BooksForm />
+    <BooksList />
+  </main>
```
Comment (taiwo2) [inline src/components/App.js]: Kindly wrap with `div` tag

File: src/containers/BooksForm.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+};
+
+const BookForm = () => (
+  <div>
+    <input type="text" id="title" placeholder="Title" />
+    <label htmlFor="categories">
+      Category:
+      <select id="categories">
+        {renderCategories(categories)}
+      </select>
```
Comment (taiwo2) [inline src/containers/BooksForm.js:16]: options for different categories is not displaying properly

**Review by taiwo2 — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED ♻️ 
@omar-labana  Nice job done on this milestone, I commend your effort in this milestone project. however few changes are required, also find my inline comments
Git flow 👍🏻 
Nice project structure 👍🏻 

#### Improve

* [x] it renders form with categories select box.  categories to select options is not working properly. kindly fix
* [x] Kindly wrap your `App.js` with tag `div`

### once specific changes are made request for another review

**Review by george-shammar — APPROVED** [review]
Hi @omar-labana , 

## Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

:heavy_check_mark: You've made the required changes.
## Congratulations! 🎉 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**